README.md

```diff
@@ -19,18 +19,36 @@ If you are not using CocoaPods, you can integrate the Placed.framework and Place
 
 ## Location Permission
 
-This SDK requires Apple's background location permission. Under the "Required background modes" key in your app’s main plist file you should make sure to add:
-
-    App registers for location updates
-
-In addition, for iOS 11, you must provide all of the following location usage descriptions in your app's plist:
-
-    NSLocationAlwaysAndWhenInUseUsageDescription
-    NSLocationWhenInUseUsageDescription
-    NSLocationAlwaysUsageDescription
-    NSLocationUsageDescription
-
-**These permissions are very important to the Placed SDK. If your app does not currently use background location permission, please contact a Placed representative.**
+As outlined in Section 5 of the [Placed Affiliate Agreement](https://affiliate.placed.com/placed-affiliate-agreement/), you must satisfy two requirements prior to registering a user with the Placed SDK:
+
+1. *Gather Express Consent for User Data Collection via Opt-in Dialog*  
+In addition having a legally compliant privacy policy describing Placed’s collection of location and device information, you must include a discrete opt-in dialog which gathers express consent for data collection. This dialog appears prior to the fine location permission prompt, and includes:  
+    - The language: “*Aggregated device data, including location, is measured for the purposes of market research by Placed, Inc.*”
+    - Links to the Placed [Terms of Service](https://www.placed.com/terms-of-service) and [Privacy Policy](https://www.placed.com/privacy-policy)
+    - Buttons to “Accept” or “Cancel” 
+
+
+1. *Prompt for Background Location Permission*  
+Under the “Required background modes” key in your app’s main plist file, you will need to add:  
+    - **App registers for location updates**   
+    - (for iOS 11) **NSLocationAlwaysAndWhenInUseUsageDescription**
+    - (for iOS 11) **NSLocationWhenInUseUsageDescription**  
+    - (for iOS 11) **NSLocationAlwaysUsageDescription**  
+    - (for iOS 11) **NSLocationUsageDescription**  
```

Comment:
Are these all for ios 11? Don't we use some of these pre 11?

Reply:
I didn't come up with this text, don't know, thought it was already discussed.

Reply:
I would vote to remove the "iOS 11" text because they'll be needed for iOS 12 and so on.